.storybook/preview.js

```diff
@@ -51,7 +45,11 @@ const App = ({ story }) => <Store>{WrappedStory(story)}</Store>;
 // Consideration for the future - using addon-redux could bring value.
 // It wasn't implemented to begin with because it wasn't compatible with Storybook 6.
 export const decorators = [
-  Story => <><App story={Story} /></>
+  (Story) => (
```

Comment:
This was a Prettier (format on save) change. However, it should be the same result.